torch/jit/_monkeytype_config.py

```diff
@@ -16,6 +15,25 @@
 except ImportError:
     _IS_MONKEYTYPE_INSTALLED = False
 
+def get_type(type):
+    """
+    Helper function which converts the given type to a torchScript acceptable format.
+    """
+    if inspect.getmodule(type) == typing:
+        # If the type is a type imported from typing
+        # like Tuple, List, Dict then replace `typing.`
+        # with a null string. This needs to be done since
+        # typing.List is not accepted by TorchScript.
+        type_to_string = str(type)
+        return type_to_string.replace(type.__module__ + '.', ' ') + ','
+    elif 'torch' in type.__module__:
```

Comment:
Maybe it is better to check that `type.__module__` starts with `torch` instead of contains `torch`?